### DISCUSSION TITLE
Slack trigger | Format message

### DISCUSSION THREAD

**Discussion by manupm87** [argoproj/argo-events#1173] [Q&A]
I'm trying to format the message sent to the slack channel and haven't yet found a way. I'm not sure if this is doable at all making use of slack triggers.

**Comment by whynowy** [comment]
Are you saying you want to make some format changes in the Sensor before sending them to Slack? Please take a looks at https://github.com/argoproj/argo-events/blob/master/docs/tutorials/02-parameterization.md#sprig-templates

**Reply by manupm87** [reply]
I'm mostly interested in formatting the slack message itself. 

See https://app.slack.com/block-kit-builder/

**Reply by kavishdahekar** [reply]
> I'm mostly interested in formatting the slack message itself.
> 
> See https://app.slack.com/block-kit-builder/

Was anybody able to figure this out?
